java/java.editor.base/src/org/netbeans/modules/java/editor/base/semantic/SemanticHighlighterBase.java

```diff
@@ -871,7 +871,18 @@ public Void visitClass(ClassTree tree, Void p) {
             }
             scan(tree.getExtendsClause(), null);
             scan(tree.getImplementsClause(), null);
-
+            try{
+            while(!tl.currentToken().text().toString().equals("{")) {
```

Comment:
Although I'm not an expert of this area, maybe, if you would like to check whether text of a token is the same as given one, should use `org.netbeans.api.lexer.TokenUtilities.textEquals()` instead of `toString().equals` (e.g. `!TokenUtilities.textEquals(tl.currentToken().text(), "{")`), then should add `// NOI18N` to strings.
Please reformat your changes. (the changes below, too)

Reply:
It doesn't seem to be reformatted yet. Please add `// NOI18N` for strings that are not expected translation.